include/zephyr/net/mqtt.h

```diff
@@ -348,13 +348,13 @@ struct mqtt_sec_config {
 	/** Indicates the list of ciphers to be used for the session.
```

Comment:
Please change the commit subject to `net: mqtt: change cipher and sec_tag lists to const` as we place the subsystem identifier at the beginning of the subject.